twindle-cli/index.js

```diff
@@ -22,21 +22,22 @@ async function main() {
 
   try {
     // this next line is wrong
-    let tweets = require("./twitter/twitter_responses/response-version2-tweetthread.json");
+    let tweets = require("./twitter/twitter_responses/mock.json");
```

Comment:
It's older version. Check out the newer one I've set in here https://github.com/twindle-co/twindle/pull/670/files#diff-b8b40d717b22ccff448e8db3c82e3999f603aa60f40304f940bf7831b294609eR25